packages/json-rpc-engine/src/index.ts

```diff
@@ -1,6 +1,18 @@
-export * from './createAsyncMiddleware';
-export * from './createScaffoldMiddleware';
-export * from './getUniqueId';
-export * from './idRemapMiddleware';
-export * from './JsonRpcEngine';
-export * from './mergeMiddleware';
+export type {
+  AsyncJsonRpcEngineNextCallback,
+  AsyncJsonrpcMiddleware,
```

Comment:
optional: Maybe we should fix this here.

```suggestion
  AsyncJsonRpcMiddleware,
```

Reply:
We really should, but I can do that in another PR.